pkg/image/trigger/buildconfigs/buildconfigs.go

```diff
@@ -175,9 +176,22 @@ func (r *buildConfigReactor) ImageChanged(obj runtime.Object, tagRetriever trigg
 			newSource = true
 		}
 
-		// LastTriggeredImageID is an image ref, despite the name
-		if latest == p.LastTriggeredImageID {
-			continue
+		// see if this trigger has fired previously and is recorded in status;
```

Comment:
```suggestion
		// Indicates whether this trigger has fired previously and is recorded in status.
```

Reply:
yeah `Indicates whether` does not quite work for me @rolfedh 

I'll go as far as `Determine whether`